var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -25,8 +27,30 @@ class Fzf(MakefilePackage):
 
     depends_on('go@1.11:')
 
+    variant('vim', default=False, description='Install vim plugins for fzf')
+
+    variant('github_mirrors', default=False, description='pull GO dependencies from github.com instead of specified mirror')
+
     def edit(self, spec, prefix):
+        if '+github_mirrors' in spec:
+            makefile = FileFilter('Makefile')
+
+            makefile.filter('go get -u github.com/Masterminds/glide && ' +
```

Comment:
@clellsolomon what goes wrong when this doesn't happen? Did the current mirror listed in the fzf package break (or the default mirrors for glide)? 

IMO this looks like a patch, which would be a bit easier to read than usage of `FileFilter` (in particular since you are replacing one fixed string with another).

In either case, a short explanation (as a response in the PR or as a comment in the code, ideally the latter) would be great.

Reply:
@scheibelp the mirror issue creeps in because it appears that the mirrors that are default specified by glide are blocked by firewalls on some systems I've tried the build on, but github mirrors exist for all of them.

I'm very open to alternative solutions to this problem.

Please forgive my ignorance, I was unaware of the patching capability.  I'll re-implement it with the patch option, it will be easier to read.